apps/web_delivery/lib/web_delivery/endpoint.ex

```diff
@@ -3,30 +3,17 @@ defmodule WebDelivery.Endpoint do
 
   socket "/socket", WebDelivery.UserSocket
 
-  # Code reloading can be explicitly enabled under the
-  # :code_reloader configuration of your endpoint.
-  if code_reloading? do
-    plug Phoenix.CodeReloader
-  end
+  if code_reloading?, do: plug Phoenix.CodeReloader
 
   plug Plug.RequestId
   plug Plug.Logger
 
   plug Plug.Parsers,
+    json_decoder: Poison,
     parsers: [:urlencoded, :multipart, :json],
-    pass: ["*/*"],
-    json_decoder: Poison
+    pass: ["*/*"]
 
-  plug Plug.MethodOverride
```

Comment:
We are not using **POST** HTTP methods via HTML forms to emulate **PUT**, **PATCH**, and **DELETE** methods.